rh._.exports({"0":["Duplicate form "],"1":["\n  ","\n    ","Duplicate form","\n    "," ","\n    "," ","\n    "," ","\n    ","I was able to track down the issue causing the error message.  It was due to duplicate forms in the profile security for the profile you provided.  I have removed the duplicate and will put it a task to have and additional duplicates removed from existing profiles.  But if you receive any reports of this moving forward I would look in the profile security for duplicate entries.","\n    "," ","\n    ","Good Afternoon,","\n    ","We are having a handful of users that are receiving an error message when they attempt to edit a corrective action from either the activity calendar or from within an event module.","\n    ","Attached are screenshots from one users that sees this message.","\n    ","\"Sequence contains more than one element\"","\n    ","I have asked these users to clear the cache and Flush ","DNS",", but they are still receiving this error message.","\n    ","I am able to open the corrective action and update the necessary information on my computer.","\n    ","This happened in November. I believe it had something to do with the security in the profile.","\n    ","This user has the following profile: Vegetation Manager ","Recordkeeper",".","\n    ","Let me know if you need any additional information.","\n    ","\n    "," ","\n  ","\n\n"],"id":"15"})